Fixed Python 3 compatibility issue with gdb/libpython.py handling of attribute
dictionaries.
